### ISSUE TITLE
Add groot-request-service

### ISSUE THREAD

**Issue by bcongdon** [acm-uiuc/groot#58]
Add [groot-request-service](https://github.com/acm-uiuc/groot-request-service) to the docker-compose files

**Comment by gauravdesale** [comment]
what would the port be for this. How would I go about finding the port to add they ml file. I got the other stuff from the other entries but is the port just a random four digit number? Thanks for your time

**Comment by bcongdon** [comment]
Yeah, you can just choose a random 4 digit number above 1024 that hasn't been used yet

**Comment by apache8080** [comment]
In the Dockerfile on the groot-request-service repo, I set the port to 5656.
